tox.ini

```diff
@@ -23,7 +23,7 @@ description =
 setenv =
     MPLBACKEND = agg
     SUNPY_SAMPLEDIR = {env:SUNPY_SAMPLEDIR:{toxinidir}/.tox/sample_data/}
-    PYTEST_COMMAND = pytest -vvv -r a --pyargs sunpy --cov-report=xml --cov=sunpy --cov-config={toxinidir}/setup.cfg {toxinidir}/docs
+    PYTEST_COMMAND = pytest -vvv -r fEs --pyargs sunpy --cov-report=xml --cov=sunpy --cov-config={toxinidir}/setup.cfg {toxinidir}/docs
```

Comment:
This sets it up so it tells us about failed, errored or skipped tests.

With the a, it was all but it now returns the full information on the xfailed (i swear pytest<8 did not) and seeing that just clogs up the logs.